experimental/instrumentation-performance-timeline/README.md

```diff
@@ -30,15 +19,11 @@ initializeFaro({
     ...getWebInstrumentations(),
   ],
   transports: [
-    // Add batch transport.
-    new BatchTransport(
-      // Choose the Faro transport of your choice.
-      new OtlpHttpTransport({
-        apiKey: env.faro.apiKey,
-        logsURL: 'https://example.com/v1/logs',
-        tracesURL: 'https://example.com/v1/traces',
-      })()
-    ),
+    new OtlpHttpTransport({
```

Comment:
Since Fetch transport is our default and it supports bacthing, let's chnage the example to use FetchTrasnport instead.

Reply:
I am resolving this as you fixed it  😆